pkg/test/auth/tokenmanager.go

```diff
@@ -0,0 +1,194 @@
+package test
```

Comment:
package should be `auth`

Reply:
fixed: 15dc7702d0869b27c7bd3c0a2bf96e2166c0e7f6